fimsauthz-api/fims-authz.yaml

```diff
@@ -66,19 +67,18 @@ paths:
                   type: string
                 scope:
                   description: |
-                    White space separated list of the requested scope(s).
-                    The scope request list size is usually one.  In cases where the list size is not one, the list is verified.  Version 1.0.2 of this specificaiton verifies this set: [dss.read.identification_service_areas, rid.read.enhanced_details]
-                  type: string
+                    White space separated set of the requested scope(s).
```

Comment:
Are we OK with ANY white space?  https://simple.wikipedia.org/wiki/Whitespace_(computer_science)

tabs?  returns?  I think we should be specific to avoid problems.  probably just space-character separated only?

Reply:
Will change to align with Token Request and Token Response in https://tools.ietf.org/html/rfc6749#section-3.3:
The value of the scope parameter is expressed as a list of space- delimited, case-sensitive strings.

thx @myspri  for pointing me to this spec.